doc/README.rst

```diff
@@ -26,13 +26,13 @@ their respective websites.
 The project's documentation contains the following items:
 
 * ReStructuredText source files used to generate documentation found at the
-  https://zephyrproject.org/doc website. Most of the reStructuredText sources
+  http://docs.zephyrproject.org website. Most of the reStructuredText sources
```

Comment:
We should always link to `https://` links if they're available.

Reply:
I agree. However docs.zephyrproject.org does not support https.